main.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+function createTable(id, year, month) {
+  const calBody = document.getElementById(`${id}`);
```

Comment:
calBody looks not good for me, just use calendarBody. You do not need to type it always IDE will help

Reply:
template string is overkill

```javascript
  const calBody = document.getElementById(id);
```